test/integration/api/projects.js

```diff
@@ -59,6 +59,20 @@ describe('api: /projects', () => {
                 body[0].name.should.equal('Default Project');
               }))))));
 
+    it('should return the verbs of the more permissive assignment if multiply assigned', testService((service) =>
+      service.login('alice', (asAlice) =>
+        asAlice.get('/v1/users/current').expect(200).then(({ body }) => body.id)
+          .then((aliceId) => asAlice.post('/v1/projects/1/assignments/manager/' + aliceId)
+            .expect(200)
+            .then(() => asAlice.get('/v1/projects/1')
```

Comment:
This is the existing/unchanged single project endpoint, but I wanted to add this test about the verbs themselves. Though the name may not be accurate since it's a different query doing a different thing.